tests/eas/cpuhp.py

```diff
@@ -0,0 +1,217 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2017, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os.path
+import random
+import json
+import sys
+import time
+
+from test import LisaTest, experiment_test
```

Comment:
experiment_test isn't used

Reply:
Still to be removed?

Reply:
Actually `experiment_test` is used at the bottom of the file.